sys-devel/gcc/gcc-5.4.0_2016_07_08.recipe

```diff
@@ -0,0 +1,347 @@
+SUMMARY="C/C++ compiler for target ${effectiveTargetMachineTriple}"
+DESCRIPTION="The standard compiler for non-legacy Haiku (i.e. for all \
+architectures other than x86_gcc2)."
+HOMEPAGE="http://gcc.gnu.org"
+
+SOURCE_URI="https://ftp.gnu.org/gnu/gcc/gcc-5.4.0/gcc-5.4.0.tar.bz2"
```

Comment:
It seems everyone that comes to this project has his own set of rules. It reminds me Jonathan Schleifer, who ironically has the opposite stance to yours (more secure is better).
Moreover I think the builder at hpkg.mlotz.ch is configured to only build recipes with safe sources. So I don't see any bad practice at using HTTPS FWIW.